src/containers/home/index.tsx

```diff
@@ -131,39 +130,12 @@ const Home: React.FC = () => {
           </Col>
         </Row>
         <Row align="middle">
-          <UpcomingEventsTitle>Annoucements</UpcomingEventsTitle>
-          <ViewMoreButton to="upcoming-events">
-            View All Annoucements
+          <UpcomingEventsTitle>Announcements</UpcomingEventsTitle>
+          <ViewMoreButton to="announcements">
+            View All Announcements
           </ViewMoreButton>
         </Row>
-        <Row gutter={[24, 24]} justify="center">
-          <Col>
-            <AnnoucementCard
-              src="http://www.pinkgazelle.com/wp-content/uploads/2017/04/monarch_butterfly_flower.jpg"
-              title="VIRTUAL Slow Flow Restorative Yoga"
-              date={new Date()}
-              description={`Find some time for self-care and join yoga teacher Sarah Oleson
-    for a peaceful and rejuvenating virtual...`}
-            />
-          </Col>
-          <Col>
-            <AnnoucementCard
-              src="https://facts.net/wp-content/uploads/2020/07/monarch-butterfly-facts.jpg"
-              title="VIRTUAL Slow Flow Restorative Yoga"
-              date={new Date()}
-              description={`Find some time for self-care and join yoga teacher Sarah Oleson
-    for a peaceful and rejuvenating virtual...`}
-            />
-          </Col>
-          <Col>
-            <AnnoucementCard
-              title="VIRTUAL Slow Flow Restorative Yoga"
-              date={new Date()}
-              description={`Find some time for self-care and join yoga teacher Sarah Oleson
-    for a peaceful and rejuvenating virtual...`}
-            />
-          </Col>
-        </Row>
+        <AnnouncementsList limit={3} />
```

Comment:
You will also here have to connect to the redux store and dispatch the action to load events. The point is to make it clear where data lives in the application, by limiting it to container files, behavior is much easier to expect.

Reply:
Why would I connect to the redux store and dispatch here, if I'm gonna be doing it once I call Announcements? Btw, this is will now be `<Announcements limit={3} />` instead of `<AnnouncementsList limit={3} />`

Reply:
@jackblanc  thoughts? If the dispatch is in the AnnoucementsList component I dont see why we need to move it into index.tsx unless you want to make a rule that we can only call dispatch in a container?

Reply:
Yes, we should only call dispatch and access data from the Redux store in containers. Components should not contain dependencies on the store, they need to be able to be rendered independently.